app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -278,11 +278,6 @@ public static void restartCommCare(Activity activity) {
         intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP |

```

Comment:
Those require API 16 and API 21. Do you suggest  using them if the API level is met?
Also seems strange to launch an activity from the activity we are closing. Wondering if using a pending intent w/ a timer would be a good approach: http://stackoverflow.com/a/17166729